_data/configuration-navigation.yaml

```diff
@@ -0,0 +1,4 @@
+- field: "Number of visible main navigation items"
```

Comment:
I’m unsure if this is actually needed.

Reply:
We could skip it in v1 as least? What do you say @frederfred ? Do we need it for Norr?

Reply:
I thought we did. I need to see the proposed navigation design until I know for sure.

One way to go forward is to remove it for now and add it when we know for sure that we need it.

Reply:
Ok, did so now...